io/src/test/scala/sbt/io/IOSpec.scala

```diff
@@ -15,14 +16,17 @@ class IOSpec extends FlatSpec with Matchers {
     // and
     // relativeRootDir referring to io-relativize/inside-dir/../
 
-    val rootDir = Files.createTempDirectory("io-relativize")
+    val rootDir = Files.createTempDirectory("io-relativize").normalize
     val nestedFile = Files.createFile(rootDir resolve "meh.file").toFile
     val nestedDir = Files.createDirectory(rootDir resolve "inside-dir").toFile
 
     val relativeRootDir = new File(nestedDir, "..")
-
-    IO.relativize(rootDir.toFile, nestedFile).map(file) shouldBe Some(file("meh.file"))
-    IO.relativize(relativeRootDir, nestedFile).map(file) shouldBe Some(file("meh.file"))
+    assert(
+      IO.relativize(rootDir.toFile, nestedFile)
+        .map(normalizeForWindows) === Option("meh.file"))
```

Comment:
Is the change from `Some` to `Option` to fix type inference?

Reply:
I am not sure if `Option(...)` change was necessary for ScalaTest's `===`, but many `Eq` typeclasses prefer the types to line up exactly on lhs and rhs.